packages/create-gatsby/scripts/import-options-schema.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+const path = require("path")
+const fs = require("fs-extra")
+const pluginPath= process.argv[2]
+const Joi  = require("joi")
```

Comment:
```suggestion
const Joi  = require("gatsby-plugin-utils")
```

To make sure any extensions we add to Joi are usable!